README.md

```diff
@@ -200,6 +200,8 @@ For information about the governance of the Node.js project, see
   **Michaël Zasso** <<targos@protonmail.com>> (he/him)
 * [tniessen](https://github.com/tniessen) -
   **Tobias Nießen** <<tniessen@tnie.de>> (he/him)
+* [ShogunPanda](https://github.com/ShogunPanda) -
+  **Paolo Insogna** <<paolo@cowtech.it>> (he/him)
```

Comment:
I think `ShogunPanda` should appear between `ruyadorno` and `targos`, no?

Reply:
@aduh95 thanks, I guess I need to work on my ABCs :) Fixed.